docset/winserver2022-ps/netsecurity/New-NetFirewallRule.md

```diff
@@ -689,6 +689,15 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -PolicyAppId 
+Specifies WDAC policy tag that matches with this rule.  The tag can contain any alphanumeric characters and any of the characters ":", "/", ".", and "_". The tag should match the AppIdTags specified in a WDAC tagging policy and so it would be in the process token security attributes of the tagged applications this rule would be enforced on.
```

Comment:
```suggestion
Specifies the WDAC policy tag that matches this rule.  The tag can contain any alphanumeric characters and any of the characters ":", "/", ".", and "_". The tag should match the AppIdTags specified in a WDAC tagging policy and so it would be in the process token security attributes of the tagged applications this rule would be enforced on.
```